### ISSUE TITLE
optionally pull back crossrefs in bibliographies. 

### ISSUE THREAD

**Issue by kohlhase** [brucemiller/LaTeXML#800]
This is related to #766, and the styling of bibliographies. In the application of using LaTeXML to generate publication nice lists I only have one more problem: I do not want ```crossref``` to result in a citation reference. For instance, if I run the following document through latexml
```
\documentclass{article}
\bibliography{foo}
\begin{document}
\nocite{BanKoh:mmlof07}
\end{document}
```
with the following bibTeX database ```foo.bib```
```
@InCollection{BanKoh:mmlof07,
   author = {Grzegorz Bancerek and Michael Kohlhase},
   title = {Towards a {Mizar Mathematical Library} in {OMD}oc Format},
   pages = {265--275},
   url = {http://kwarc.info/kohlhase/papers/trybook.pdf},
   crossref = {MatZal:fip07},
pubs = {mkohlhase}}

@Book{MatZal:fip07,
    title = {From Insight to Proof: Festschrift in Honour of {Andrzej Trybulec}},
    booktitle = {From Insight to Proof: Festschrift in Honour of {Andrzej Trybulec}},
    series = {Studies in Logic, Grammar and Rhetoric},
    editor = {Matuszewski, R. and Zalewska, A.},
    publisher = {University of Bia{\l}ystok},
    year = {2007},
    volume = {10:23},
    url={http://mizar.org/trybulec65/}}
```
currently generates (note the ```See 21```). 
```
[4] R. Matuszewski and A. Zalewska (Eds.) (2007) From insight to proof: festschrift in honour of Andrzej Trybulec. Studies in Logic, Grammar and Rhetoric, Vol. 10:23, University of Białystok. External Links: Link 
[11] G. Bancerek and M. Kohlhase Towards a Mizar Mathematical Library in OMDoc format. See 21, pp. 265–275. External Links: Link
```
There are two problems in this: 
* the numbering is wrong
* I would really like to see 
```
[11] G. Bancerek and M. Kohlhase Towards a Mizar Mathematical Library in OMDoc format. In R. Matuszewski and A. Zalewska (Eds.) (2007) From insight to proof: festschrift in honour of Andrzej Trybulec. Studies in Logic, Grammar and Rhetoric, Vol. 10:23, University of Białystok. External Links: Link,  pp. 265–275. External Links: Link
```
It would be very nice if we could have a way for LaTeXML to inline the crossrefs in processing. 
This could be an option, but it could also be a macro I give in the source file, e.g. ```\nobibcrossrefs``` or something similar.

**Comment by kohlhase** [comment]
Even though I think that LaTeXML should solve this problem, I will try to work around this for my application: https://github.com/KWARC/bibs/issues/7

**Comment by kohlhase** [comment]
Hmmm, after the "let's dump this on Bruce" reflex wore off, I realized that this is really easy to do in a postprocessing step. I made a little [XSLT style sheet](https://raw.githubusercontent.com/KWARC/bibs/master/src/ltxml/crossrefs.xsl) that does the trick. It does however make some assumptions (see the comments), and may not work for you. 
So instead of 
```
latexmlc --bibtex --destination=foo.bib.xml foo.bib
latexmlc bar.tex --bibliography foo.bib.xml 
``` 
you simply have to do 
```
latexmlc --bibtex --destination=foo.bib.xml --stylesheet=crossrefs.xsl foo.bib
latexmlc bar.tex --bibliography foo.bib.xml 
```
